gm4_rope_ladders/data/gm4_rope_ladders/tags/blocks/air.json

```diff
@@ -0,0 +1,6 @@
+{
+  "values": [
+    "minecraft:air",
+    "minecraft:cave_air"
+  ]
+}
```

Comment:
Why does this file exist? This seems like a leftover and would be redundant as `#gm4:air` handles this for you.